server/events/project_command_builder_test.go

```diff
@@ -608,6 +608,160 @@ projects:
 	}
 }
 
+func TestDefaultProjectCommandBuilder_BuildPlanAllCommands(t *testing.T) {
```

Comment:
Nice test! Could you also create a separate test for the new function `shouldUpdateRepo` ?

Reply:
The outer scope tests this already, but I will add additional tests to cover this.